notebooks/WET/WET-004/WET-004-ComCam.ipynb

```diff
@@ -0,0 +1,385 @@
+{
```

Comment:
<p>As for WET-003, would be good to clarify that it's meant to be run in an USDF (RSP) environment. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>